golang/main.go

```diff
@@ -1,7 +1,22 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"math"
+)
 
+func formula(x float64) float64 {
+	return (math.Pow(math.Pow(math.Asin(x), 2)+math.Pow(math.Acos(x), 4), 3))
+}
 func main() {
-	fmt.Println("Hello world")
+	fmt.Println("Прибылов Алексей Александрович")
+	fmt.Println("Задача A")
+	for i := 0.26; i <= 0.66; i += 0.08 {
+		fmt.Println("При x=", i, "y=", formula(i))
```

Comment:
используйте форматированный вывод - примерно так
```suggestion
		fmt.Printf("При x=%d y=%d", i, formula(i))
```